releasenotes/config.yaml

```diff
@@ -33,7 +34,6 @@ sections:
   - [upgrade_visualization, Visualization Upgrade Notes]
   - [upgrade_misc, Misc. Upgrade Notes]
   - [deprecations, Deprecation Notes]
-  - [deprecations_algorithms, Algorithms Deprecations]
```

Comment:
Removing this might be problematic because reno crawls the git tree if it encounters a release not using this label it will error.

Reply:
So a section that was added once will remain forever? 🤔

Reply:
The other option is to remove the historical release notes, or edit them to use a different section name.